app/views/shared/_post_item.html.erb

```diff
@@ -32,4 +26,10 @@
             <% end %>
         </ul>
     <% end%>
+    <div class="add-comment" id="comment-form-<%= post.id %>">
+        <%= form_with model: @comment, method: :post, url: "/users/#{post.author.id}/posts/#{post.id}/comments", scope: 'comment' do |form| %>
+            <%= form.text_field :text, placeholder: "Add a comment...", class: "comment-text" %>
+            <%= form.submit "Add" %>
+        <% end %>
+  </div>
```

Comment:
Nice one there. However, to make your project more professional and follow best standard practices, kindly create a new file, `views/comments/new.html.erb` and put the form that will accept inputs for creating comments. :pray:

Reply:
@jerryowusu , thank you for the review